src/tests/ftest/deployment/basic_checkout.py

```diff
@@ -96,5 +96,10 @@ def test_basic_checkout_dm(self):
         self.ior_cmd.namespace = "/run/ior_dm/*"
         self.ior_cmd.get_params(self)
         self.ppn = self.params.get("ppn", '/run/ior_dm/client_processes/*')
-        #run datamover
-        self.run_dm_activities_with_ior("FS_COPY", True)
+
+        # create pool and container
+        pool = self.create_pool()
+        cont = self.get_container(pool, oclass=self.ior_cmd.dfs_oclass.value)
```

Comment:
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'oclass' in method call